scripts/lowercase-all-records.js

```diff
@@ -23,10 +22,10 @@ async function subscribeLowercaseHashToHIBP(emailAddress) {
   const subsWithUpperCount = subRecordsWithUpperChars.length;
   console.log(`found ${subsWithUpperCount} subscribers records with primary_email != lower(primary_email). fixing ...`);
   for (const subRecord of subRecordsWithUpperChars) {
-    const lowerCasedSha1 = await subscribeLowercaseHashToHIBP(subRecord.primary_email);
+    const emailHash = await subscribeLowercaseHashToHIBP(subRecord.primary_email);
     await knex("subscribers")
       .update({
-        primary_sha1: lowerCasedSha1,
+        primary_sha1: emailHash.toLowerCase(),
```

Comment:
Because we didn't explicitly lowercase hashes before storing them in our db when we last ran `lowercase-all-records.js`, does this potentially mean we may not be able to correctly match them since we lowercase all hashes when they come in from HIBP?